src/mgr/BaseMgrModule.cc

```diff
@@ -663,6 +663,43 @@ get_latest_unlabeled_counter(BaseMgrModule *self, PyObject *args)
       svc_name, svc_id, counter_path);
 }
 
+static PyObject*
+get_latest_counter(BaseMgrModule *self, PyObject *args)
+{
+  char *svc_name = nullptr;
+  char *svc_id = nullptr;
+  char *counter_name = nullptr;
+  char *sub_counter_name = nullptr;
+  PyObject *labels_list = NULL; //labels = [("level", "deep"), ("pooltype", "ec")]
```

Comment:
NULL??

Reply:
thanks for catching this, Fixed here: https://github.com/ceph/ceph/pull/61997/commits/8f8b7759ec0eeb75fcd5f585b66ae60e4e63425d#diff-418325979416216a8730e1e2bb75ab0f24c998ca6e8f4899162125b39fe315c4R673